### PR TITLE
Adjust pMA source.

### LINKED ISSUE #117: Switch from whence we get phpMemcachedAdmin
As per https://github.com/Varying-Vagrant-Vagrants/VVV/issues/589, with Google Code going away, our source for pMA is also going away. The VVV guys have helpfully [forked and re-hosted](https://github.com/wp-cloud/phpmemcacheadmin) pMA at Github (with the original author's consent).

Let's switch HGV to provision it from there, assigning to myself.

### PR DESCRIPTION
**Pull request by zamoose**
* [x] @zamoose
* [x] @markkelnar 

Change where we're pulling phpMemcachedAdmin from (since Google Code is going away).

Fixes #117.

### CODE REVIEW

**Comment by markkelnar** [comment]
I ran into an issue with an error "already exists and is not an empty directory", when running this on an existing vagrant where /nas/wp/www/sites/admin/phpmyadmin exists from the url download and not as a .git clone.

How to get around this @zamoose?

**Comment by zamoose** [comment]
This will only be a problem for existing installs. We can either document the solution ("delete hgv_data/sites/admin/phpmyadmin") or we can put an explicit `stat`/`register` check in the task. I'd advocate for documentation, as it would be a long-term fix to a short-term problem.

**Comment by markkelnar** [comment]
But who is really going to read the documentation?  Will it be frustrating for users to encounter the error, have to dig, then finally reach out because we broke backward compatibility?  For that, I lean a little toward blowing the dir if the /.git doesn't exist in there.

**Comment by markkelnar** [comment]
Or, I guess the user would have to run the vagrant provision in order to invoke the ansible run.  This wouldn't be experienced on a regular 'vagrant up' on an existing instance.  So, maybe it's less critical of a problem?

**Comment by zamoose** [comment]
That's kind of what I'm thinking. Hmmm. Let me noodle this over for a
second.

On Wed, Apr 22, 2015 at 4:36 PM, Mark Kelnar <notifications@github.com>
wrote:

> Or, I guess the user would have to run the vagrant provision in order to
> invoke the ansible run. This wouldn't be experienced on a regular 'vagrant
> up' on an existing instance. So, maybe it's less critical of a problem?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/wpengine/hgv/pull/123#issuecomment-95327642>.
>



-- 
-Doug

File: provisioning/roles/admin/tasks/main.yml [hunk 1]
```diff
@@ -38,16 +38,31 @@
 #
 # phpMemcachedAdmin setup
 #
+- name: Check whether pMA is a Git checkout
```
Comment (zamoose) [inline provisioning/roles/admin/tasks/main.yml]: @markkelnar How's this look to you for the purposes of checking and deleting?

**Comment by markkelnar** [comment]
:+1:

**Pull request merged**